cg/services/order_validation_service/rules/case_sample/rules.py

```diff
@@ -327,3 +329,15 @@ def validate_concentration_interval_if_skip_rc(
         )
         errors.extend(case_errors)
     return errors
+
+
+def validate_case_sample_well_position_format(
```

Comment:
I do not mind naming this just `validate_well_position_format` (and potentially have the same name for the sample rule as well. I think this is a bit clunkier.